### DISCUSSION TITLE
Please accept the [Amazon Partnered Carrier Program] terms and conditions in country: [DE]

### DISCUSSION THREAD

**Discussion by IncredibleX** [abuzuhri/Amazon-SP-API-CSharp#150] [Q&A] [answered]
Hello,

I have been working on the documentation in detail.

The goal is to send a delivery schedule and receive the labels at the end via SPAPI.

Step 1 - Submit carton content information
Step 2 - Submit the transportation information
Step 3 - Request that an estimate is generated for the shipping costs
Step 4 - Get an estimate for the shipping cost
Step 5 - Confirm the transportation request
Step 6 - Request package labels for your inbound shipment
Step 7 - Request pallet labels or bills of lading (LTL/FTL only)
Step 8 - Send your shipments to Amazon's fulfillment network

I fail at step 3 - there I get the error message. Obviously I still have to agree to the conditions. Where is it documented how I agree to the conditions via the SPAPI?
Errormessage: Please accept the [Amazon Partnered Carrier Program] terms and conditions in country: [DE]

I got the errormessage when i try to use EstimateTransport.

`    FikaAmazonAPI.AmazonSpApiSDK.Models.FulfillmentInbound.CommonTransportResult oPutTransportDetails = amazonConnection.FulFillmentInbound.PutTransportDetails(sShipmentID, oPutTransportDetailsRequest);
                FikaAmazonAPI.AmazonSpApiSDK.Models.FulfillmentInbound.CommonTransportResult oEstimation = amazonConnection.FulFillmentInbound.EstimateTransport(sShipmentID);
               `

Thank you,

Greetings

Tobias Stephan

**Comment by IncredibleX** [comment, marked as answer]
I spent some time trying to figure out how to solve the problem.
5 minutes after writing this text, I found the solution. The setting must be done in [Sellercentral](https://sellercentral.amazon.de/fba/settings/index.html/ref=xx_fbasettings_dnav_xx#/pcp-contract-status).

**Reply by RenzoF** [reply]
Hi, thanks for posting this answer here, is this still working ok for you?
I am receiving the following error and all the T&C's are accepted for all countries.
Please accept the [null] terms and conditions in country: [GB]

**Reply by GWTorstenDEV** [reply]
Hi, you need to accept the Terms & Condition of the shipping vendor at sellercentral. e.g.

pls sign in to your sellercental account, upper right corner (settings) go under "Fulfilment by Amazon" than the part "Amazon Partnered Carrier (Small Parcel Deliveries) - MWS Settings". click edit

UPS Terms and Conditions must be accepted from the country your sending from.

done

greets
Torsten
